### DISCUSSION TITLE
Adding collection variables into the report

### DISCUSSION THREAD

**Discussion by prasanmgc** [DannyDainton/newman-reporter-htmlextra#408] [Ideas]
Gone through the documentation but haven't found a way to show the collection variables in the report. Am I missing anything or the feature isn't implemented yet?

**Comment by DannyDainton** [comment]
It's not been implemented.

**Reply by prasanmgc** [reply]
Thanks @DannyDainton. Any plans in implementing them? Also Is there any option to hide the secret variables with ***** (similar to Postman UI) in the report?

**Reply by DannyDainton** [reply]
Not immediate plans from me - feel free to give it a go at adding them and send me a PR ;) 

No option to mask a variable but you can use a flag to exclude the whole thing so others don't see it in the report.

![image](https://user-images.githubusercontent.com/17089546/226566784-f8fb90e7-88ca-489a-bb40-98ac728980ce.png)
